src/main/java/marquez/api/Namespace.java

```diff
@@ -0,0 +1,75 @@
+package marquez.api;
```

Comment:
I think we are moving non-request/response entities to `core/models`?

Reply:
I think so, but I'd rather make that refactoring change when we move all the entities over (e.g. Owner, JobRun, etc.).